python/tests/extras/test_image_metric.py

```diff
@@ -39,27 +44,45 @@ def image_loader(path: str = None) -> ImageType:
         return img
 
 
+IMAGE_METRIC_CONFIG = ImageMetricConfig(
+    exif_tags={
```

Comment:
what is an exif tag?

Reply:
Exchangeable Image File Format metadata, available in a variety of image formats.